test/command.js

```diff
@@ -85,3 +85,88 @@ test('execaCommandSync()', t => {
 	const {stdout} = execaCommandSync('node test/fixtures/echo.js foo bar');
 	t.is(stdout, 'foo\nbar');
 });
+
+test('$', async t => {
+	const {stdout} = await $`node test/fixtures/echo.js foo bar`;
+	t.is(stdout, 'foo\nbar');
+});
+
+test('$ accepts options', async t => {
+	const {stdout} = await $({encoding: 'utf8'})`node test/fixtures/echo.js foo bar`;
```

Comment:
We might want to use an example where the options changes `stdout`. The `encoding` does not change `stdout` here, so the test would pass even if the `encoding` option was ignored.

Reply:
Nice catch! For now, I just updated the tests to [properly validate the APIs accept options](https://github.com/sindresorhus/execa/pull/510/commits/0cc0fc3aa6072763252ebd664f7abfefe5e6a023), but agree we should include tests to validate different `stdout`s 👍